app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,44 @@
+class InvitationsController < ApplicationController
+
+  before_action :require_user, only: [:new, :create]
+
+  def new
+    @invitation = Invitation.new
+    @invitation.user = current_user
+  end
+
+  def create
+    params = invitation_params
```

Comment:
If you don't notice this line, this has the potential to lead to wasted hours of debugging and wondering why `params` doesn't contain a value it should. I would recommend removing this and using the explicit `invitation_params` throughout the method code.